google/cloud/bigtable/examples/run_examples_utils.sh

```diff
@@ -274,15 +284,27 @@ function run_all_table_admin_examples {
       "${project_id}" "${INSTANCE}" "${TABLE}" "max-age-family"
   run_example ./table_admin_snippets update-gc-rule \
       "${project_id}" "${INSTANCE}" "${TABLE}" "max-age-family"
-  run_example ./table_admin_snippets wait-for-consistency-check "${project_id}" "${INSTANCE}" "${TABLE}"
-  run_example ./table_admin_snippets generate-consistency-token "${project_id}" "${INSTANCE}" "${TABLE}"
+
+  run_example ./table_admin_snippets wait-for-consistency-check \
+      "${project_id}" "${INSTANCE}" "${TABLE}"
+  run_example ./table_admin_snippets generate-consistency-token \
+      "${project_id}" "${INSTANCE}" "${TABLE}"
   local token
-  token="$(./table_admin_snippets generate-consistency-token "${project_id}" "${INSTANCE}" "${TABLE}" | awk '{print $5}')"
-  run_example ./table_admin_snippets check-consistency "${project_id}" "${INSTANCE}" "${TABLE}" "${token}"
-  run_example ./table_admin_snippets drop-rows-by-prefix "${project_id}" "${INSTANCE}" "${TABLE}"
-  run_example ./data_snippets read-rows "${project_id}" "${INSTANCE}" "${TABLE}"
-  run_example ./table_admin_snippets drop-all-rows "${project_id}" "${INSTANCE}" "${TABLE}"
-  run_example ./table_admin_snippets delete-table "${project_id}" "${INSTANCE}" "${TABLE}"
+  token="$(./table_admin_snippets generate-consistency-token \
```

Comment:
Need we call `generate-consistency-token` twice?

Reply:
Need?  No.  I do like to see this output (when I look at the output that is):

```
[ RUN      ]  table_admin_snippets wait-for-consistency-check running
[       OK ]  table_admin_snippets wait-for-consistency-check
```

Which does not appear if all we do is capture the return in `token="$(...)"`.

That is, I would prefer to keep it.

Reply:
Ack.